XRTK-Core/Packages/com.xrtk.core/Definitions/BaseMixedRealityExtensionServiceProfile.cs

```diff
@@ -5,8 +5,7 @@
 
 namespace XRTK.Definitions
 {
-    public abstract class BaseMixedRealityExtensionServiceProfile : BaseMixedRealityServiceProfile<IMixedRealityService>
+    public abstract class BaseMixedRealityExtensionServiceProfile : BaseMixedRealityServiceProfile<IMixedRealityExtensionDataProvider>
```

Comment:
Note: The generic type argument here is the type we wish to filter our configurations profile's list by